lukimgather/schema.py

```diff
@@ -0,0 +1,17 @@
+import graphene
+from django.conf import settings
+from graphene_django.debug import DjangoDebug
+
+from user.schema import UserMutations, UserQueries
+
+
+class Query(UserQueries, graphene.ObjectType):
+    if not settings.DEBUG:
```

Comment:
Enable for non debug?
```suggestion
    if settings.DEBUG:
```